package.json

```diff
@@ -25,7 +25,7 @@
         "moment": "^2.24.0",
         "mydatepicker": "^9.0.2",
         "ngx-bootstrap": "^4.3.0",
-        "ngx-quill": "^5.2.0",
+        "ngx-quill": "^12.0.1",
```

Comment:
Laut [Kompatibilitätsübersicht](https://github.com/KillerCodeMonkey/ngx-quill#compatibility-to-angular-versions) sollte hier maximal v11 installiert werden

Reply:
Gibt auch "unmet peer dependency"-Warnungen:
![Bildschirmfoto 2020-10-28 um 08 44 20](https://user-images.githubusercontent.com/26869118/97406647-d4243c80-18f9-11eb-8310-b9d6fc4d51a8.png)